libs/coin-modules/coin-mina/src/api/index.integ.test.ts

```diff
@@ -0,0 +1,69 @@
+import { getAccount, getBlockInfo, getTransactions } from ".";
+import { fetchNetworkStatus } from "./rosetta";
+import { getCoinConfig } from "../config";
+import { CurrencyConfig } from "@ledgerhq/coin-framework/config";
+
+jest.mock("../config");
+describe("backend api tests", () => {
+  jest.mocked(getCoinConfig).mockReturnValue({
+    ...({} as unknown as CurrencyConfig),
+    infra: {
+      API_MINA_ROSETTA_NODE: "https://mina.coin.ledger.com/node",
+    },
+  });
+  const validAddress = ["B62qjWLs1W3J2fFGixeX49w1o7VvSGuMBNotnFhzs3PZ7PbtdFbhdeD"];
+  const invalidAddress = [
+    "B62qkdFWJSW8zaTBZjTVtmeU3rVxyUkNxPhKKW8T2JBtpj5XfdywLJM",
+    "novalidaddress",
+  ];
+
+  it("getAccount for valid address", async () => {
+    const account = await getAccount(validAddress[0]);
+    expect(account).toBeDefined();
+    expect(account.balance.toNumber()).toBeGreaterThan(0);
+  });
+
+  it("get network status", async () => {
+    const networkStatus = await fetchNetworkStatus();
```

Comment:
This method is implemented in rosetta, maybe we can move it to rosetta/index.integ.test.ts ? or maybe move all tests in rosetta.integ.test.ts here